tensorflow_datasets/image/imagewoof.py

```diff
@@ -0,0 +1,141 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Imagewoof a subset of 10 easily classified classes from Imagenet.
+(Australian terrier, Border terrier, Samoyed, Beagle, Shih-Tzu, 
+English foxhound, Rhodesian ridgeback, Dingo, Golden retriever, 
+Old English sheepdog)
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import tensorflow as tf
+import tensorflow_datasets.public_api as tfds
+
+# TODO(Imagewoof): BibTeX citation
+_CITATION = """
+"""
+
+_DESCRIPTION = """\
```

Comment:
Hi @WilliamHYZhang,  
Thanks for your review! I had not included testing/fake_data_utils in this pull request because of my local repo space issues. I will be including them into follow-up. Good catch!!

I didn't understand your other comment about the following statement...."Note that afterwards...??

A quick suggestion to you -  all these instructions are well written in the documentation provided in **Add datasets...**
https://www.tensorflow.org/datasets/add_dataset#1_add_an_import_for_registration

So please direct anyone who is asking help, it is well written and very easy to follow!!

Good luck to you too!!

Reply:
I agree that the documentation is quite well written, it also helped me a lot when I was working on these tasks. Please see my other comment about the "Note that afterwards..." :)